nestkernel/connection_creator_impl.h

```diff
@@ -337,6 +392,82 @@ ConnectionCreator::pairwise_bernoulli_on_target_( Layer< D >& source,
   }
 }
 
+
+template < int D >
+void
+ConnectionCreator::pairwise_poisson_( Layer< D >& source,
+  NodeCollectionPTR source_nc,
+  Layer< D >& target,
+  NodeCollectionPTR target_nc )
+{
+  // Connect using pairwise Poisson drawing source nodes (target driven)
+  // For each local target node:
+  //  1. Apply Mask to source layer
+  //  2. For each source node: Compute probability, draw random number, make
+  //     connection conditionally
+
+  // retrieve global positions, either for masked or unmasked pool
+  PoolWrapper_< D > pool;
+  if ( mask_.get() ) // MaskedLayer will be freed by PoolWrapper d'tor
+  {
+    pool.define( new MaskedLayer< D >( source, mask_, allow_oversized_, source_nc ) );
+  }
+  else
+  {
+    pool.define( source.get_global_positions_vector( source_nc ) );
+  }
+
+  std::vector< std::shared_ptr< WrappedThreadException > > exceptions_raised_( kernel().vp_manager.get_num_threads() );
+
+// sharing specs on next line commented out because gcc 4.2 cannot handle them
+#pragma omp parallel // default(none) shared(source, target, masked_layer,
+                     // target_begin, target_end)
```

Comment:
We don't use sharing info on any other parallel sections, so just remove these comments (please also do so in all other places they occur in this file):
```suggestion
#pragma omp parallel
```

Reply:
Addressed in 1920cdc163f953f6bb4e44f980bb1dcfd62e507d.